packages/react-examples/src/react-charting/DonutChart/DonutChart.Dynamic.Example.tsx

```diff
@@ -15,8 +15,21 @@ export interface IExampleState {
   hideLabels: boolean;
   showLabelsInPercent: boolean;
   innerRadius: number;
+  statusKey: number;
+  statusMessage: string;
 }
 
+const screenReaderOnlyStyle: React.CSSProperties = {
+  position: 'absolute',
+  width: '1px',
```

Comment:
Could you add a description in the PR about this fix.
Why this css style.
What does aria-live and aria-atomic mean

Reply:
also the alternative aria-alert option that you explored.

Reply:
updated the PR description and added some comments